examples/client/beatBuilder.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html lang="en">
+<head>
```

Comment:
Looking good! Last thing is the html tag still hasn't been opened correctly. Take a look at the example below. Doctype tag isn't html tag. 

```html
<!DOCTYPE html>
<html>
<head>
<title>Title of the document</title>
</head>

<body>
The content of the document......
</body>

</html>
```